cmd/zpool/zpool_main.c

```diff
@@ -4285,6 +4301,15 @@ zpool_do_iostat(int argc, char **argv)
 				    gettext("Can't set -c flag twice\n"));
 				exit(1);
 			}
+
+			if (getenv("ZPOOL_SCRIPTS_ENABLED") != NULL &&
+			    !libzfs_envvar_is_set("ZPOOL_SCRIPTS_ENABLED")) {
+				fprintf(stderr, gettext(
+				    "Can't run -c unless "
+				    "ZPOOL_SCRIPTS_ENABLED is set.\n"));
```

Comment:
The wording in the error message is misleading since it implies you must have this set for -c to work. Instead how about this, "Can't run -c, disabled by ZPOOL_SCRIPTS_ENABLED".